libafl/src/feedbacks/map.rs

```diff
@@ -525,16 +525,16 @@ where
         }
 
         let initial = observer.initial();
-        if interesting {
```

Comment:
This has some speed impact, right

Reply:
yes this part is the "Danger zone". it's the no.1 hot path in libafl

Reply:
More recent changes show a ~5-10% speedup in map feedback.

Reply:
nice 💯